src/Frame.js

```diff
@@ -0,0 +1,7 @@
+'use strict'
+
+function Frame(nr, rolls, score) {
```

Comment:
A couple questions:

What behaviour does this object have?

What should it be responsible for?